### DISCUSSION TITLE
Problem with exporting tracks

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#340] [Q&A] [answered]
occurs in b4 version:
Exporting a track generates a 0 byte file. In Android 13 I got a window like attempt to invoke... I couldn't read it that fast and it doesn't appear again, it just writes a zero byte file, same as in the other phone with Android 10.

**Comment by WolfgangZ42** [comment]
No problem here - export/import with .json works in b4, Android 10.

**Comment by r11d11** [comment]
I had the same issue with a 0 Byte exported file some days ago, I think I happens with b3. But it happens only one time. So i did not report it.
WIth b4 and Android 13 I just exported json, gpx und kurviger to my cloud and all where saved correct,
I exported it to my download folder on my phone and it works. 
I fear the problem will not be easy to find...

**Comment by devemux86** [comment]
@DolomitesTom thanks for the report.

Can you mention how do you export the track?
Is it with the new favorite tracks screen in GPX?

Can you reproduce it every time?

**Reply by DolomitesTom** [reply]
Yes I can reproduce it any time. It is tapping in top left navigation bar with recorded track kilometers, then export track as gpx.


The same error occours when exporting from the new tracks menu

![Screenshot_20231007-152313_Gallery](https://github.com/devemux86/cruiser/assets/121371629/ae5b963b-89a3-49fe-bab5-208d2b181072)

**Reply by devemux86** [reply]
@DolomitesTom thanks for the extra details.

I can reproduce the error in the export of GPS recording. It will work properly again in next Cruiser update.

**Reply by devemux86** [reply]
I can also reproduce the 2nd report, it will work properly in next Cruiser update.

@DolomitesTom thanks for catching these issues! :+1:

**Comment by devemux86** [comment]
It should work properly again with the latest:
- https://github.com/devemux86/cruiser/discussions/329

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
